tests/System.Buffers.Primitives.Tests/MemoryTests.cs

```diff
@@ -71,6 +71,24 @@ static void TestLifetime(OwnedBuffer<byte> owned)
         }
 
 
+        [Fact]
+        public void TestThrowOnAccessAfterDipose()
+        {
+            var array = new byte[1024];
+            OwnedBuffer<byte> owned = array;
```

Comment:
Many are in samples or experimental. I will add tests for other implementations in System.Buffer.Primitives.